Civi.php

```diff
@@ -95,6 +95,7 @@ public static function service($id) {
   public static function reset() {
     self::$statics = array();
     Civi\Core\Container::singleton();
+    \Civi::service('settings_manager')->useDefaults();
```

Comment:
Agree it's really confusing. IMHO, the `$force` idiom is at least consistent with other stuff, but `$loadFromDB==FALSE` is really weird. (If I had to guess, it probably exists because some common utility function like `ts()` or `escapeString()` turns batty without `CRM_Core_Config`.)

When refactoring before, I figured we needed to retain it on general principle, but now I'm not so sure: grepping universe (`rgrep CRM_Core_Config::singleton | grep -v 'singleton()'`), there are only a handful of cases which use `$loadFromDB==FALSE`. Feel like there's 2/3 chance we could kill that mode with 1-2 days work.